frontend/test/tickets/tickets.store.spec.ts

```diff
@@ -31,36 +31,37 @@ describe('Tickets: store', () => {
 		({ dispatch, getState } = createTestStore());
 	});
 
-	it('should fetch and set model tickets', () => {
-		const ticket = ticketMockFactory();
-		dispatch(TicketsActions.fetchTicketsSuccess(modelId, [ticket]));
-		const modelTicketsFromState = selectTickets(getState(), modelId);
-
-		expect(modelTicketsFromState[0]).toEqual(ticket);
-	});
-
-	it('should update the model tickets', () => {
-		const ticket = ticketMockFactory();
-		dispatch(TicketsActions.fetchTicketsSuccess(modelId, [ticket]));
-
-		const oldTicket = cloneDeep(ticket)
-		const modifications = { _id: ticket._id, title:'modified ticket', properties:{priority:'Top'}}
+	describe('templates', () => {
```

Comment:
Not a big deal, but this describe seems to contain `tickets` stuff rather than `templates` (which is the name of the next describe)

Reply:
Ah yeah, well spotted. I'll fix it